src/sbvr-api/sbvr-utils.coffee

```diff
@@ -703,7 +717,7 @@ exports.handleODataRequest = handleODataRequest = (req, res, next) ->
 				req.hooks = {}
```

Comment:
We can avoid the `Promise.resolve()` in the lines above by duplicating the `tapCatch`